nautobot/core/graphql/generators.py

```diff
@@ -354,7 +360,7 @@ def generate_attrs_for_schema_type(schema_type):
     # Define Attributes for single item and list with their search parameters
     search_params = generate_list_search_parameters(schema_type)
     attrs[single_item_name] = graphene.Field(schema_type, id=graphene.ID())
-    attrs[list_name] = graphene.List(schema_type, **search_params)
+    attrs[list_name] = DjangoListField(schema_type, **search_params)
```

Comment:
[DjangoListField](https://github.com/graphql-python/graphene-django/blob/main/graphene_django/fields.py#L21-L22) has a positional argument called `_type` as well. That is why we are running into this error.